src/cgi/mfs.cgi.in

```diff
@@ -3219,6 +3219,7 @@ if "HELP" in sectionset:
     # print_file("@CGI_PATH@/help.html")
     print("""please contact with help@lizardfs.com""")
     print("""<br/>""")
+    print("""<iframe src="https://docs.lizardfs.com/", width="100%", height="100%"></iframe>""")
```

Comment:
```suggestion
    print("""<iframe src="https://docs.lizardfs.com/" width="100%" height="100%"></iframe>""")
```
I just noticed that you have another mistake in your code: `,` aren't used to separate HTML attributes.